README.md

```diff
@@ -34,6 +34,9 @@ Check out [the docs](http://hexdocs.pm/ex_machina/ExMachina.html) for more detai
 Define factories:
 
 ```elixir
+# You could put this in `test/support/factory.ex` if you use it for tests only,
```

Comment:
One problem with this is that I believe that `test/support` is not in the `elixirc_paths` by default. In phoenix it is, but a regular mix project doesn't include that as far as I know. Any ideas for that? Maybe something like

```
# You could put this in `test/factory.ex` or in `lib/my_app/factory.ex` if you want to use it for development seed data as well.
# If using Phoenix, you may want to put it in `test/support/factory.ex`